6/build website

AdamVig opened the pull request:
Closes #6.

AdamVig committed:
Remove old site and prepare for new site

AdamVig committed:
Add back container from Skeleton grid

AdamVig committed:
Add nav bar

AdamVig committed:
Replace logo text with image

This will ensure that the logo is always represented in the correct
font and with the correct letter spacing.

AdamVig committed:
Replace Skeleton button primary colors

AdamVig committed:
Fix nav title vertical centering

AdamVig committed:
Add header

AdamVig committed:
Add call to action

AdamVig committed:
Add content to sections

AdamVig committed:
Add equipment overhead hero image

This required removing the container class from the `main` tag so that
individual `section`s inside the `main` could be the full width of the
page.

AdamVig committed:
Add background image properties mixin

This will prevent repetition when creating other classes with
background images that are intended to fit in the flow of the page.

AdamVig committed:
Increase paragraph font size of whole page instead of just header

AdamVig committed:
Add footer styles

AdamVig committed:
Add images and layout to content sections

- Add more space below header
- Move equipment overhead hero image to bottom of content section
- Replace team section with equipment overhead hero
- Add padding to all sections

AdamVig committed:
Shorten header copy

AdamVig committed:
Fix photography kit styles and layout

- Remove duplicate classes
- Change from fixed to percentage height to match text block height
- Remove column offset to allow side-by-side display on mobile

AdamVig committed:
Center header content on small screens

AdamVig committed:
Move nav styles into a separate file

This will allow the nav styles to be imported into the specific Sass
file for each additional page.

AdamVig committed:
Add subscribe page placeholder

AdamVig committed:
Fix missing photography kit image on larger screens

AdamVig commented:
*Note: Screenshots are for reference only! They are not a very good representation of the website.*
# Homepage
![image](https://user-images.githubusercontent.com/5354752/26999411-b40a7b74-4d66-11e7-8b73-2ac4a4c9985a.png)
![image](https://user-images.githubusercontent.com/5354752/26999433-6fbff6b4-4d67-11e7-845d-28e7a053310a.png)
![image](https://user-images.githubusercontent.com/5354752/26999439-88be2a00-4d67-11e7-9213-c7c71fc13b42.png)


# Subscribe Page
![image](https://user-images.githubusercontent.com/5354752/26999406-98884b42-4d66-11e7-8dda-dc165978fb32.png)

emroussel commented on src/index.html:
```diff
@@ -3,33 +3,104 @@
+            </section>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <div class="stockpile-inventory"></div>
```
Should this be `<div class="stockpile-scan"></div>` instead?

emroussel commented on src/index.html:
```diff
@@ -3,33 +3,104 @@
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <p>
+                        Create kits of items that go together so you don't forget
+                        anything when you rent items.
```
I think reading through the different sections, the word 'item' is repeated a lot. Is there a way we can rephrase some of them to make it flow a little better?

emroussel reviewed (changes requested):
Great job!

I think in general I would add more space between every sections and between the header and the first section.

Also, I'm not sure if this was voluntary or not, but the website as a whole feels off centered. The sections in the middle are aligned to the right. There is also more space between the text and the image in the section for scanning than there is for the section about viewing inventory and kits.

Finally, I feel like screenshots would look better if they are placed inside a phone frame.

AdamVig committed:
Fix wrong image class in scan section

The class was `stockpile-inventory` instead of `stockpile-scan` due to
a copy-paste error.

AdamVig committed:
Remove offset from content columns

The offset was originally intended to push the two columns closer
together, but backfired by offsetting the entire content section to
the right side.

AdamVig committed:
Add phone frame to screenshots

AdamVig committed:
Move screenshot height to variable

AdamVig committed:
Center background images in their containers

This fixes the tipi at the top of the page being off-center on smaller
screens.

AdamVig committed:
Add size parameter to background image mixin

This allows the equipment overhead image to use the mixin, because it
has a different `background-size` property than the rest of the
classes that use it, so it was hard-coded in before.

AdamVig committed:
Change order of equipment overhead class to match page layout

AdamVig committed:
Add more padding between nav and header

AdamVig committed:
Decrease hero font size on small screens

The text and background were totally obscuring the image behind them.

AdamVig committed:
Remove excessive use of the word "item"

AdamVig committed:
Add space around header

AdamVig committed:
Fix photography kit image missing on small screens

AdamVig committed:
Fix gap on right side of page

Using `border-box` box sizing prevents accidentally creating elements
that are wider than the viewport when their width is combined with
padding and margins. Hiding the overflow of the `html` and `body`
elements gets rid of the ~5px gap on the right side of the page that
could be scrolled over to.

AdamVig committed:
Fix tipi offset to the right on small screens

AdamVig committed:
Remove empty space from screenshots

AdamVig committed:
Add more space around text in sections

AdamVig committed:
Improve responsive layout

- Center content (text and images) on small screens
- Put images above text on small screens
- Add a class for elements with background images
- Decrease size of photography kit image

AdamVig committed:
Change breakpoints to variables

AdamVig committed:
Reduce size of Stockpile logo in nav

AdamVig commented:
Awesome feedback, thanks! I realized as soon as I looked at it again that it needed a lot more work. 

I think I addressed each of your pieces of feedback. If I missed anything, please tell me!

Here are updated screenshots, which are actually accurate this time (I used an extension instead of hand-capturing them 😆).

### Laptop with HiDPI Screen
![image](https://user-images.githubusercontent.com/5354752/27007620-3106d184-4e27-11e7-8255-7944b8783354.png)

### iPad
![image](https://user-images.githubusercontent.com/5354752/27007622-39965630-4e27-11e7-8362-8936ebfeb5ea.png)

### iPhone 6
![image](https://user-images.githubusercontent.com/5354752/27007623-3dc6d680-4e27-11e7-9966-a2247090eb56.png)

emroussel commented:
Yes it looks much better, good job!

Two more things that I thought of when looking at it again:
- Maybe adding a title for each section would be helpful to indicate which image/screenshot is associated with what text on mobile (One or very few words describing what the section is about). The whitespace helps, but I still had a few moments of hesitation when I first saw it. It could be above the text or centered above both the image and text, not sure which would look better. Could be similar to [stripe.com](https://stripe.com/) if you think it fits for Stockpile.
- If you had titles for sections, it might make sense to have a navigation bar in the header that has links to these different sections.

Also, I just realized that we never mention that what we are selling is an app 😂

Might be worth adding that to the description or somewhere else. Another option might be to have the classic two images of iOS and Android with our app instead of where the logo is right now and move the logo to the navbar or something.

AdamVig committed:
Add headings to content sections

AdamVig commented:
### 1080p Screen
![image](https://user-images.githubusercontent.com/5354752/27016280-1ea096cc-4eec-11e7-8e5e-3bdcd8857951.png)

### iPhone 6
![image](https://user-images.githubusercontent.com/5354752/27016316-73dc0b94-4eec-11e7-8aba-683714a05096.png)

I decided to pass on adding section links to the nav bar. The page is so short and simple that I don't think section links would add any value. I will add links to the subscribe page and the barcodes page once they exist, but that will be in each of their pull requests, not this one.

I thought about how to add the word "app" in somewhere, but couldn't see any place where it fit naturally. I think the screenshots do a good enough job of getting the point across. We'll have to do some user testing of our website copy, though, to check if it actually communicates well.

emroussel commented:
Sounds good, and I love the titles of sections!
